docs/commands/graph.query.md

```diff
@@ -885,79 +885,89 @@ This section contains information on all supported functions from the Cypher que
-| toStringList()    | Converts a list of values to a list of string values. Each item in the list is converted using toStringOrNull()        | 
-| toBoolean()       | Returns a boolean value from boolean type, string type (either `"True"` or `"False"` string values, case insensitive. All other values will return `null`) or integer type (0 for `false`, all other values will return `true`). All other types are not allowed and will cause a type mismatch error to be returned.
-| toBooleanOrNull() | Returns a boolean value from boolean type, string type (either `"True"` or `"False"` string values, case insensitive. All other values will return `null`) or integer type (0 for `false`, all other values will return `true`). All other types will return `null`.
-| toBooleanList()   | Converts a list of values to a list of boolean values. Each item in the list is converted using toBooleanOrNull()      |
+|Function                     | Description|
+| --------------------------- |:-----------|
+| toBoolean(_expr_)           | Returns a Boolean value unchanged <br> Converts a string to Boolean (`"true"` (case insensitive) to true, `"false"` (case insensitive) to false, any other value to null) <br> Converts an integer to Boolean (0 to `false`, any other values to `true`) <br> Returns null when _expr_ is null <br> Error on other types |
+| toBooleanList(_exprList_)   | Converts a list of values to a list of Booleans. Each element in the list is converted using toBooleanOrNull()                                                                                             |
+| toBooleanOrNull(_expr_)     | Returns a Boolean unchanged <br> Converts a string to Boolean (`"true"` (case insensitive) to true, `"false"` (case insensitive) to false, any other value to null) <br> Converts an integer to Boolean (0 to `false`, any other values to `true`) <br> Returns null when _expr_ is null <br> Returns null for other types   |
+| toFloat(_expr_)             | Returns a floating point unchanged <br> Converts an integer to a floating point <br> Converts a string to a floating point or null <br> Returns null when _expr_ is null <br> Error on other types         |
+| toFloatList(_exprList_)     | Converts a list of values to a list of floating points. Each element in the list is converted using toFloatOrNull()                                                                                        |
+| toFloatOrNull(_expr_)       | Returns a floating point unchanged <br> Converts an integer to a floating point <br> Converts a string to a floating point or null <br> Returns null when _expr_ is null <br> Returns null for other types |
+| toInteger(_expr_) *         | Returns an integer unchanged <br> Converts a floating point to integer <br> Converts a string to an integer or null <br> Converts a Boolean to an integer (false to 0, true to 1) <br> Returns null when _expr_ is null <br> Error on other types         |
```

Comment:
`Returns an integer unchanged` consider rephrasing,
e.g. returns `expr` unchanged if `expr` is an Integer

This suggestion applies to other functions with similar documentation.

Reply:
`expr` can be 2+3, so "return `expr` unchanged" is actually wrong.
Changing to "When `expr` evaluates to an integer: returns its evaluation"